cap-2.x/eks/variables.tf.json

```diff
@@ -8,74 +8,75 @@
         "instance_type": {
             "type": "string",
             "description": "The type of instance used for the provisioned workers.",
-            "default": "Standard_DS3_v2"
```

Comment:
Github says `cap-2.x/aks/cluster/variables.tf.json → cap-2.x/eks/variables.tf.json` ... so is cap-2.x/aks/cluster/variables* not required?

Reply:
No, the top-level provider directories (aks/eks/gke) all have `variables.tf.json`.  Some of the sub-directories and modules therein have their own `variables.tf` but not all do. `eks/cluster` is just a parent directory for other EKS cluster "sub-modules".